### ISSUE TITLE
[Suggestion] Wiki documentation

### ISSUE THREAD

**Issue by claunia** [qmfrederik/dotnet-packaging#103]
It would be great to have a wiki documentation explaining how .csproj/.fsproj attributes and elements map to package fields.

Right now it is particularly difficult to even know how to fill things like:

Debian package section.
Package conflicts.
Package "provides".
Et al.

**Comment by etursunbaev** [comment]
I agree, it would be nice if there will be some detailed documentation based on wiki.
I use this project to build deb package, but met lack information
Something where I can define control properties, post and pre install scripts and etc.

**Comment by AnthonyMastrean** [comment]
In the meantime, you can get a lot of info from `Package.Targets.targets` https://github.com/qmfrederik/dotnet-packaging/blob/master/Packaging.Targets/build/Packaging.Targets.targets
